M2/Macaulay2/packages/PushForward.m2

```diff
@@ -184,7 +184,7 @@ document{
   A = kk[a,d]
   use R
   F = map(R,A)
-  pushFwd F
+  stack(net \ pushFwd F)
```

Comment:
That looks pretty artificial.  I recommend this instead:
```m2
i15 : (M,g,pf) = pushFwd F;

i16 : M

o16 = cokernel {0} | 0  |
               {1} | 0  |
               {2} | -d |
               {1} | 0  |
               {2} | a  |

                             5
o16 : A-module, quotient of A

i17 : g

o17 = | 1 b b2 c c2 |

              1       5
o17 : Matrix R  <--- R
```

Reply:
That way we don't even have to look at the function.  Extending the example code to demonstrate the use of the function might be worthwhile.

Reply:
Yes, that is way better!  I've just fixed up this commit using your suggestion.